### DISCUSSION TITLE
Error while deploying "/usr/bin/git config --get remote.origin.url"

### DISCUSSION THREAD

**Discussion by Base29** [deployphp/deployer#3381] [Help needed]
Hello guys I am having issues while deploying my laravel application ... Couple of deployments went well. Still, today when I tried to deploy my updated code I saw this error `run cd /var/www/mysite/backend/.dep/repo && (/usr/bin/git config --get remote.origin.url)` on `deploy:update_code` ... please guide on this issue. What is the best solution for it? 

UPDATE:

I have commented on the code block on line 95 located in `vendor/deployer/deployer/recipe/deploy/update_code.php` but it seems to be still running the git command  `/usr/bin/git config --get remote.origin.url`

Seems to me that the command is cached or something.

<img width="742" alt="image" src="https://user-images.githubusercontent.com/5220908/201375785-47bbc703-8ad2-4764-9c41-31eff073f293.png">

**Comment by Base29** [comment]
Any suggestions?

**Comment by Base29** [comment]
For the time being there is a workaround for it. You just need to delete `.dep/repo` folder on the server.

**Reply by leighgregg** [reply]
Thanks, was this logged as an error? I had the issue recently.
